pilot/pkg/xds/endpoints/endpoint_builder.go

```diff
@@ -689,17 +663,14 @@ func buildEnvoyLbEndpoint(b *EndpointBuilder, e *model.IstioEndpoint, mtlsEnable
 			return nil
 		}
 		// For inbound, we only use EDS for the VIP cases. The VIP cluster will point to encap listener.
-		if supportsTunnel {
```

Comment:
we build inbound-vip cluster only for waypoint, so no need to check here
@kyessenov to take a look

Reply:
what about `e.SupportsTunnel`? That is for the destination, not the client

Reply:
It might be OK to assume the server is "ambient" here. Waypoint only makes sense when the server traffic is captured.

Reply:
Yeah that's what i think, normally we build `inbound-vip|xxxx` cluster for services that are in scope for waypoint.

There is rare case that an endpoint can be excluded, so i will revert this protect.